app/Landing.js

```diff
@@ -12,14 +12,12 @@ class Landing extends Component {
     return (
       <View style={{
           flex: 1,
-          flexDirection: 'column',
-          justifyContent: 'space-around',
-          alignItems: 'center'
+          margin: 8
         }}>
         <Logo/>
-        <FindRepsButton/>
-        <CallingScriptsButton/>
-        <TipsStrategiesButton/>
+        <LoadSceneButton text='Find My Reps' />
+        <LoadSceneButton text='Calling Scripts' />
```

Comment:
This makes a ton of sense! I was digging through the docs to make this work and that's why I came up with the three different files. Much better solution here. Thank you!